source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1408,9 +1445,47 @@ def isochrone(self, request, instance):
         # Initialize a context for distributed
         distributed_context = self.get_context()
         request_id = request.get("request_id", None)
+
+        pt_object_origin = None
+        pt_object_destination = None
+        if request.get('origin'):
+            origin_detail = self.get_entrypoint_detail(
```

Comment:
The logic of using `get_entry_detail`, then using `json_address_from_uri` as plan B is already done line [1212](https://github.com/hove-io/navitia/pull/3970/files#diff-6f8087c75aa4667ea11ebca99cccdd3f0d228658d83a1d6d13bf58b375a0e21dR1212). Maybe there is an opportunity for factorizing code here.

Reply:
Done